.github/workflows/workflow.yml

```diff
@@ -127,3 +127,26 @@ jobs:
           SSH_CONFIG: ${{ secrets[env.SSH_CONFIG_SECRET_NAME] }}
           SSH_KEY: ${{ secrets[env.SSH_KEY_SECRET_NAME] }}
         run: ./.github/scripts/deploy.sh
+
+      - name: Login
+        if: env.TIER == 'PROD'
+        uses: atlassian/gajira-login@master
+        env:
+          JIRA_BASE_URL: ${{ secrets.JIRA_BASE_URL }}
+          JIRA_USER_EMAIL: ${{ secrets.JIRA_USER_EMAIL }}
+          JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
+
+      - name: Create Jira Transition List
+        if: env.TIER == 'PROD'
+        run: curl https://terraware.github.io/terraware-web/unreleased.log | grep -E 'SW-[0-9]+' -o | sort -u > ./docs/jiralist.txt
+
+      - name: Transition Jiras
+        if: env.TIER == 'PROD'
+        uses: terraware/gajira-transition-multiple@master
+        with:
+          issueList: ./docs/jiralist.txt
+          transition: 'Released to Production'
+
+      - name: Clean up Jira files
+        if: env.TIER == 'PROD'
+        run: rm ./docs/jiralist.txt
```

Comment:
missed this - not sure if the cleanup is needed, alternatively can create the file under `/tmp` and skip cleanup

Reply:
Removed the cleanup